ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -125,7 +125,7 @@ public void navigateTo(String username, String title) {
 		allLinks.find(Condition.exactText(title)).shouldBe(visible).click();
 	}
 
-	@Given("^clean application state$")
+	//@Given("^clean application state$")
 	public void resetState() {
```

Comment:
Can't this be removed completely?

Reply:
See the disclaimer :) I'd keep it a bit until new impl proof itself :)